test/acceptance/tests.py

```diff
@@ -275,15 +301,19 @@ def test_staff_assessment(self):
         # Verify staff grade section appears as expected
         self._verify_staff_grade_section("NOT AVAILABLE", "WAITING FOR A STAFF GRADE")
 
-        # TODO: as part of @cahren's work on TNL-3493, change this section to do a proper full staff grade
-        # The override is a temporary hack for acceptance test advancement.
+        # Perform staff assessment
         self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
-        self.do_staff_override(username)
+        self.do_staff_assessment()
 
-        # Switch back to original user, verify staff grade section appears as expected
+        # Verify staff grade section appears as expected
         self.staff_asmnt_page.visit()
-        self.staff_asmnt_page.verify_status_value("COMPLETE")
+        self._verify_staff_grade_section(self.STAFF_GRADE_EXISTS, None)
+        self.assertEqual(self.EXPECTED_SCORE, self.grade_page.wait_for_page().score)
 
+        # Verify that staff scores can be overriden
+        self.do_staff_override(username)
+        self.browser.refresh()
+        self.assertEqual(self.STAFF_OVERRIDE_SCORE, self.grade_page.wait_for_page().score)
```

Comment:
It seemed simplest to just add this check on as another assertion on the simple `test_staff_assessment`, but it could also get it's own test method if we think that's necessary.